pkgs/development/python-modules/async-dns/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, poetry
+, python
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "async-dns";
+  version = "1.1.9";
+  disabled = pythonOlder "3.6";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "gera2ld";
+    repo = "async_dns";
+    rev = "v${version}";
+    sha256 = "0hrpjgbssvr95z9jdnmx853r6h6mih0w138hz301ll94mi8zpbsg";
+  };
+
+  nativeBuildInputs = [
+    # Switch to poetry-core, https://github.com/gera2ld/async_dns/pull/22
```

Comment:
Can we fetch that PR?

Reply:
And add a remark to remove for async-dns>1.1.9

Reply:
Use patch for now.